server/emits.py

```diff
@@ -76,31 +76,53 @@ def update_handler(sid, data=None):
 # Different effects for Co-Op and AI modes
 def setup_varied_updates(sio, rooms_tracker):
     gesture = {'data': None}
+    model = nn.NeuralNetworkBlocksworldModel({
+        'flips': 'flips.h5',
+        'colors': 'colors.h5',
+        'letters': 'letters.h5'
+    })
 
     def gesture_handler(sid, data):
         if is_coop(data):
             roommate_emit(sio, sid, 'update_gesture_data', rooms_tracker, data)
         else:
             gesture['data'] = data
 
-    model = nn.NeuralNetworkBlocksworldModel('./model.h5')
     def user_message_handler(sid, data):
         if is_coop(data):
             room_emit(sio, sid, 'update_user_message', rooms_tracker, data)
         else:
-            (position_data, movement_data) = model.generate_move(
+            move = model.generate_move(
                 sid,
                 gesture['data'],
                 data
             )
+
+            print("Received move: " + str(move))
+            if not move:
+                print("Failed to find the requested block.")
+                return
+
             gesture['data'] = None
 
-            self_emit(sio, sid, 'update_position',
-                      rooms_tracker, position_data)
-            self_emit(sio, sid, 'update_movement_data',
-                      rooms_tracker, movement_data)
-            self_emit(sio, sid, 'Update_score',
-                      rooms_tracker)
+            if move['type'] == 'flip':
+                self_emit(sio, sid, 'update_flip_block',
+                          rooms_tracker, move['block_id'][1:])
```

Comment:
Can we leave (a) comment(s) on what `move['block_id']` is and why we're taking various substrings? I've also created #25 to clean this up later.